src/webserver/WsBase.h

```diff
@@ -44,20 +45,28 @@ getDefaultWsResponse(boost::json::value const& id)
     return defaultResp;
 }
 
-class WsBase
+class WsBase : public util::DefaultTaggable
```

Comment:
We need to be able to tag each individual request coming over a websocket. Websockets can handle multiple requests, either in parallel or sequentially, and we need to be able to distinguish each of those requests from each other.